dicoogle/src/main/java/pt/ua/dicoogle/plugins/PluginController.java

```diff
@@ -83,7 +83,8 @@ public synchronized static PluginController getInstance() {
     private final WebUIPluginManager webUI;
     private final DicooglePlatformProxy proxy;
     private TaskManager taskManager = new TaskManager(Integer.parseInt(System.getProperty("dicoogle.taskManager.nThreads", "4")));
-    
+    private TaskManager taskManagerQueries = new TaskManager(Integer.parseInt(System.getProperty("dicoogle.taskManager.nThreads", "4")));

```

Comment:
We should let us specify the thread pool size separately from the other task manager.

Reply:
Yes, forgot to add after our meeting.